### PR TITLE
Initial commit of work based off digital-register-frontend

### PR DESCRIPTION
**Pull request by nick-gravgaard**

### CODE REVIEW

**Comment by russwillis** [comment]
All good. Seen it running

File: service/api_client.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import json
```
Comment (lgonczar) [inline service/api_client.py]: We probably don't need this file anymore.

File: service/address_utils.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
+import re
```
Comment (lgonczar) [inline service/address_utils.py:1]: Do we still need this module?

**Comment by lgonczar** [comment]
Reviewed. Looks good, left two questions.

File: service/address_utils.py (hunk 2 above)
Comment (nick-gravgaard) [inline service/address_utils.py:1]: Yep

File: service/api_client.py (hunk 1 above)
Comment (nick-gravgaard) [inline service/api_client.py]: Agreed.

**Pull request merged**